README.md

```diff
@@ -77,3 +77,10 @@ Kubernetes: `>=1.15.0-0`
 | web.service.port | int | `80` |  |
 | web.service.type | string | `"ClusterIP"` |  |
 | web.tolerations | list | `[]` |  |
+| web.livenessProbe.enabled | bool | `true` | |
```

Comment:
This can be generated using the `.github/helm-docs.sh` script. This is also the reason for CI to fail on this repo.